src/fetcher.ts

```diff
@@ -601,7 +601,11 @@ class N3Handler extends Handler {
   ): ExtendedResponse | Promise<FetchError> {
     // Parse the text of this N3 file
     let kb = fetcher.store
-    let p = N3Parser(kb, kb, options.original.value, options.original.value,
+    let baseUrl = options.original.value
+    const isContainer = kb.any(options.original, null, ns.ldp('Container'))
+    console.log('@@ isContainer ' + isContainer)
+    if (isContainer && !baseUrl.endsWith('/')) baseUrl = baseUrl + '/'
```

Comment:
I don't think this is the proper way to solve this problem, what if the redirect goes to a different path altogether (for whatever reason)? We should be reading from the `response.url` instead. And if we don't consider redirecting to a completely different resource a valid situation, throw an error rather than keep going with potentially wrong data.